addons/sys_external/fnc_externalRadioPFH.sqf

```diff
@@ -0,0 +1,48 @@
+/*
+ * Author: ACRE2Team
+ * Eventhandler to monitor if external radios can be still used by a specific player.
+ *
+ * - Disabled uses of external radios are:
+ *   - Radio is in players inventory.
+ *   - Player is dead. External radios are returned to the original owner.
+ *   - Distance between owner and end user is greater than 2m.
+ *   - Owner and end user are not in the same vehicle.
+ * - Radios in dead or captive units can always be accessed by other players.
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [] call acre_sys_external_externalRadioPFH
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+/* TODO:
+ * - Consider further corner cases.
```

Comment:
No, I may want to add something depending on the feedback. It is just a reminder.